docs/multisite/multisite_configuration.md

```diff
@@ -49,6 +49,7 @@ and ideally also separate Varnish/Fastly configuration.
 !!! caution
 
     Different SiteAccesses can only have different `var_dir` if they also have different repositories.
+    Make sure you don't have special or Unicode characters in your `var_dir` values.
```

Comment:
```suggestion
    Make sure there are no special or Unicode characters in  your `var_dir` values.
```

Reply:
Additional space ("in  your...") not needed.

Reply:
Additional space not needed.